### PR TITLE
docs: improve setup instructions for js-example

### PR DESCRIPTION
**Pull request by hbmartin**
## Minor documentation editing

_Please excuse my laziness in not filling out the below_

To see the changes rendered: https://github.com/hbmartin/maxGraph/tree/main/packages/js-example

### CODE REVIEW

File: packages/js-example/README.md [hunk 1]
```diff
@@ -5,14 +5,22 @@ Demonstrate how to import/decode an XML model using `Codecs`.
 ## Setup
 
 Initialize all packages
-> From the repository root, run `npm install`.
+From the repository root, run:
+```bash
+npm install
+cd packages/core/
+npm pack
+cd ../../packages/example
```
Comment (tbouffard) [inline packages/js-example/README.md]: **suggestion:** wrong path
```suggestion
cd ../../packages/js-example
```

File: packages/js-example/README.md [hunk 2]
```diff
@@ -5,14 +5,22 @@ Demonstrate how to import/decode an XML model using `Codecs`.
 ## Setup
 
 Initialize all packages
-> From the repository root, run `npm install`.
+From the repository root, run:
+```bash
+npm install
+cd packages/core/
+npm pack
```
Comment (tbouffard) [inline packages/js-example/README.md]: **thought:** runinng `npm run build` is enough.
There is no need to build the npm package tar.gz file.

**Review by tbouffard — CHANGES_REQUESTED** [review]

**Comment by hbmartin** [comment]
Updated thanks!

**Review by tbouffard — APPROVED** [review]
Thanks for your contribution.

**Pull request merged**